lib/containers/postgres.dart

```diff
@@ -12,12 +24,45 @@ Future<DockerProcess> startPostgres({
   String? pgDatabase,
   int pgPort = 5432,
   bool? cleanup,
+  String? postgresqlConfPath,
+  String? pgHbaConfPath,
+  List<String>? configuraions,
```

Comment:
`configuraions` -> `configurations` (also in dartdoc)

Reply:
Done -- I should have a spell check turned on next time 😅